theme.js

```diff
@@ -16,6 +16,18 @@ const theme = {
     sm: "960px",
     md: "1280px",
     lg: "1920px"
+  },
+  color: {
```

Comment:
what country do you think we live in?!

Reply:
@maxneuvians we need your sober (or not) judgement here.

Reply:
https://github.com/HashanP/postcss-spiffing

Reply:
but the colour names are spot on